test/package.json

```diff
@@ -87,7 +88,12 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run eslint:fix"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "**/*.js": [
+      "npm run eslint:fix"
```

Comment:
I'm happy enough with us running the linter and blocking a local commit on failure. I'm a bit concerned about running the autofix and committing something the user didn't expect. Are we sure this is what we want to do here?

Reply:
I'm happy to disable the autofix, so we are clearer on what we're committing

It doesn't affect people whose IDEs run eslint autofix on save, but I've made the change anyway